AirCasting/CoreData/SessionEntity.swift

```diff
@@ -18,7 +18,9 @@ public class SessionEntity: NSManagedObject, Identifiable {
     @nonobjc public class func fetchRequest() -> NSFetchRequest<SessionEntity> {
         return NSFetchRequest<SessionEntity>(entityName: "SessionEntity")
     }
-
+    
+    @NSManaged public var peripheralUUID: String?
```

Comment:
That's a bit fishy since this property only makes sense for certain type of sessions (for a mic session it doesn't make any sense).

Currently I cannot tell you what the correct design would look like since there are moving parts here, you can try to come up with a solution or we can schedule a call to work on it together (we can also skip and just accept that technical debt, it's not a huge deal)